src/components/Image/SImage/SImage.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="s-image__container">
+    <s-skeleton
+      v-if="hasSkeleton"
+      :loading="!this.elImage || this.elImage.loading"
+      :animated="animated"
+    >
+      <template #template>
+        <s-skeleton-item element="image" />
+      </template>
+      <template>
+        <el-image
+          ref="image"
+          class="s-image"
+          :src="src"
+          :fit="fit"
+          :alt="alt"
+          :lazy="lazy"
+          :z-ndex="zIndex"
+        />
+      </template>
+    </s-skeleton>
+    <el-image v-else
+      ref="image"
+      class="s-image"
+      :src="src"
+      :fit="fit"
+      :alt="alt"
+      :lazy="lazy"
+      :z-ndex="zIndex"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Ref } from 'vue-property-decorator'
+import ElImage from 'element-ui/lib/image'
+
+import { SSkeleton, SSkeletonItem } from '../../Skeleton'
+import { ImageFit } from '../consts'
+
+@Component({
+  components: {
+    ElImage,
+    SSkeleton,
+    SSkeletonItem
+  }
+})
+export default class SImage extends Vue {
+  /**
+   * Image source, same as native
+   */
+  @Prop({ default: null, type: String }) readonly src!: string | null
```

Comment:
it should be required as I see it, without default. Btw, `() => null`. You cannot set objects, arrays and null like this